jmclient/jmclient/wallet_utils.py

```diff
@@ -43,26 +44,20 @@ def get_wallettool_parser():
                       dest='showprivkey',
                       help='print private key along with address, default false')
     parser.add_option('-m',
-                      '--maxmixdepth',
+                      '--mixdepth',
                       action='store',
                       type='int',
-                      dest='mixdepths',
-                      help='how many mixing depths to initialize in the wallet',
-                      default=5)
+                      dest='mixdepth',
+                      help="Mixdepth to use in the wallet. Default: {}"
```

Comment:
This is the wallet tool parser, so don't you think 'mixdepth to use in the wallet' is not a good description of -m here? That would be appropriate for sendpayment, but not here, right? Here it's the maximum mixdepth to display, isn't it?

Reply:
This setting is used for creating a new wallet (thus indicating how many mixdepths to initialise), for specifying the mixdepth to import a private key to and for specifying how many mixdepths to consider in display[all]

Reply:
In the first and third cases, a plural mixdepth*s* is appropriate; only in the hopefully very exceptional case, which I think we should not encourage (importing), are we talking about one mixdepth.

I get that a brief/general "to use in the wallet" makes sense to cover all cases, though. Would prefer 'mixdepths'.

But this opens another issue: there is a method 'importprivkey' but it is not included in the help message for the wallet-tool script. Let's add a line for that help message also. (Or like I prefer, kill key imports with fire! :) )